ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DeploymentComment.scala

```diff
@@ -4,9 +4,11 @@ import cats.data.Validated
 import cats.data.Validated.{Invalid, Valid}
 import pl.touk.nussknacker.ui.listener.Comment
 
-class DeploymentComment private(comment: String) extends Comment(comment) {
+class DeploymentComment private(comment: String) extends Comment {
```

Comment:
why not just `class DeploymentComment private(override val value: String) extends Comment {` ?

Reply:
you are right it's better like this, thanks